spec/statesman/adapters/active_record_queries_spec.rb

```diff
@@ -15,7 +15,14 @@
 
   before do
     MyActiveRecordModel.send(:include, Statesman::Adapters::ActiveRecordQueries)
+    MyActiveRecordModel.send(:has_many,
```

Comment:
:+1: on the change, but I'd do this is a separate `context` block. That way it will be more explicit that we're testing this case in particular and no overzealous refactoring will change these specs :)